File: Problem/Easy/Problem_2/Solution.cpp
// C++ program print numbers from N to 1 without using loops

#include <iostream>

// Recursive function
void print1ToN(int n)
{
    if (n < 1)
        return;
    std::cout << n << " ";
    print1ToN(n - 1);
}

// Main function
int main()
{
    int n = 10;
    print1ToN(n);
    return 0;
}

File: Problem/Easy/Problem_2/Solution.py
# Python3 program print N to 1 without loop

# Recursive function
def printNTo1(n : int) -> None:
    if n < 1:
        return;
    print(n, end=' ')
    printNTo1(n - 1)

# Main function
if __name__ == "__main__":
    n = 10
    printNTo1(10)